package.json

```diff
@@ -12,6 +12,7 @@
     "@types/pizzip": "^3.0.2",
```

Comment:
@RobertPorterfield -  You should up the TMCR version to 0.1.4, and add a CHANGELOG entry for it.  I know Github keeps track of these things, but it provides a nice quick look into the history of the development